wrims/bnd.bnd

```diff
@@ -0,0 +1,137 @@
+    org.eclipse.ui.ide,\
+    org.eclipse.ui.part,\
+    org.eclipse.ui.texteditor,\
+    org.osgi.framework
+
+Export-Package: \
+    wrimsv2_plugin.debugger,\
+    wrimsv2_plugin.debugger.breakpoint,\
+    wrimsv2_plugin.debugger.commanditem,\
+    wrimsv2_plugin.debugger.core,\
+    wrimsv2_plugin.debugger.exception,\
+    wrimsv2_plugin.debugger.launcher,\
+    wrimsv2_plugin.debugger.menuitem,\
+    wrimsv2_plugin.debugger.model,\
+    wrimsv2_plugin.debugger.sourcelookup,\
+    wrimsv2_plugin.debugger.toolbaritem,\
+    wrimsv2_plugin.debugger.ui,\
+    wrimsv2_plugin.debugger.view,\
+    wrimsv2_plugin.presentation,\
+    wrimsv2_plugin.tools
+
+-includeresource: \
+  plugin.xml = plugin.xml,\
+  icons = icons,\
+  lib_x64/wrims-core.jar=build/tmp/libs/wrims-core-${project.version}.jar,\
+  lib_x64/jep.dll=build/tmp/libs/jep.dll,\
+  lib_x64/antlr-runtime-3.5.2.jar=build/tmp/libs/antlr-runtime-3.5.2.jar,\
+  lib_x64/coinor.jar=build/tmp/libs/coinor.jar,\
+  lib_x64/com.google.ortools.linearsolver.jar=build/tmp/libs/com.google.ortools.linearsolver.jar,\
+  lib_x64/commons-io-2.1.jar=build/tmp/libs/commons-io-2.1.jar,\
+  lib_x64/dom4j-1.6.1.jar=build/tmp/libs/dom4j-1.6.1.jar,\
+  lib_x64/data.jar=build/tmp/libs/data.jar,\
+  lib_x64/dsm2-input-model.jar=build/tmp/libs/dsm2-input-model.jar,\
+  lib_x64/guava-11.0.2.jar=build/tmp/libs/guava-11.0.2.jar,\
+  lib_x64/gurobi.jar=build/tmp/libs/gurobi.jar,\
+  lib_x64/iText.jar=build/tmp/libs/iText.jar,\
+  lib_x64/jarh5obj.jar=build/tmp/libs/jarh5obj.jar,\
+  lib_x64/jarhdf5-2.11.0.jar=build/tmp/libs/jarhdf5-2.11.0.jar,\
+  lib_x64/jarhdfobj.jar=build/tmp/libs/jarhdfobj.jar,\
+  lib_x64/java-object-diff-0.95.jar=build/tmp/libs/java-object-diff-0.95.jar,\
+  lib_x64/javatuples-1.2.jar=build/tmp/libs/javatuples-1.2.jar,\
+  lib_x64/jep-3.8.2.jar=build/tmp/libs/jep-3.8.2.jar,\
+  lib_x64/jfreechart-1.0.19.jar=build/tmp/libs/jfreechart-1.0.19.jar,\
+  lib_x64/jnios.jar=build/tmp/libs/jnios.jar,\
+  lib_x64/jpy.jar=build/tmp/libs/jpy.jar,\
+  lib_x64/kryo-2.24.0.jar=build/tmp/libs/kryo-2.24.0.jar,\
+  lib_x64/minlog-1.2.jar=build/tmp/libs/minlog-1.2.jar,\
+  lib_x64/misc.jar=build/tmp/libs/misc-1.1.jar,\
+  lib_x64/objenesis-1.2.jar=build/tmp/libs/objenesis-1.2.jar,\
+  lib_x64/poi-3.8.jar=build/tmp/libs/poi-3.8.jar,\
+  lib_x64/poi-ooxml-3.8.jar=build/tmp/libs/poi-ooxml-3.8.jar,\
+  lib_x64/poi-ooxml-schemas-3.8.jar=build/tmp/libs/poi-ooxml-schemas-3.8.jar,\
+  lib_x64/slf4j-api-1.7.22.jar=build/tmp/libs/slf4j-api-1.7.22.jar,\
+  lib_x64/slf4j-nop-1.7.5.jar=build/tmp/libs/slf4j-nop-1.7.5.jar,\
+  lib_x64/sqljdbc4-2.0.jar=build/tmp/libs/sqljdbc4-2.0.jar,\
+  lib_x64/stax-api-1.0.1.jar=build/tmp/libs/stax-api-1.0.1.jar,\
+  lib_x64/vista.jar=build/tmp/libs/vista.jar,\
+  lib_x64/XAOptimizer.jar=build/tmp/libs/XAOptimizer.jar,\
+  lib_x64/xml.jar=build/tmp/libs/xml.jar,\
+  lib_x64/xmlbeans-2.3.0.jar=build/tmp/libs/xmlbeans-2.3.0.jar
+
+Bundle-ClassPath: \
+    lib_x64/antlr-runtime-3.5.2.jar,\
```

Comment:
What prevents this from being generated via Gradle? Seems like the parts referencing specific jars should be generated from a Gradle configuration, not hard-coded so that when the Gradle dependencies are updated, the classpath is updated too

Reply:
This will be optimized when the third party jars are extracted into the 3rd party module under the next milestone.

Reply:
After some pretty thorough investigation, it's rather non-trivial to assemble this list from Gradle with the currently available information.

I will approve this on the condition that it will be addressed as part of the 3rd party bundle, but if this manual classpath is left in for that, it will be flagged in review.